django_celery_extensions/celery.py

```diff
@@ -3,11 +3,14 @@
 from celery import Celery as BaseCelery
 
 
-class CeleryQueueEnum(Enum):
+class CeleryQueueEnum(str, Enum):
 
-    def __init__(self, queue_name, default_task_kwargs=None):
-        self.queue_name = queue_name
-        self.default_task_kwargs = default_task_kwargs or {}
+    def __new__(cls, value, default_task_kwargs):
```

Comment:
this is right way how to implement extra enum arguments according to documentation